index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>A Tribute Page.</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Dr. Norman Borlaug </h1>
+    <h4>The man who saved a billion lives </h4>
+    <img src="tribute-page-main-image.jpg"
+    alt="a lecturer with seven male students in a wheat feild"
+    height="100" width="80">
+    <br> 
+    <p> Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.
+    </p>
+    <h3> Here's a time line of Dr. Borlaug's life:</h3>
+        <ul>
```

Comment:
I think it would look nice if you could create a new div that encapsulates the list

create the new div tag and assign to it a class to be able to style the lists in your project adequately
also, the lists tags in your code will be embedded inside the div tag in order to style the lists content separately
this would give your list a kind of unique heading as seen in the real project
kindly check this link https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div to read more about the div tag and lists tags

Reply:
- [ ] Instead of framing it in this way, you could frame it like "It would be better to wrap the lists in a semantic tag as per the HTML best practices. And by giving them class for styling as per following CSS best practices."
_Good job in proving the link for reference._